README.md

```diff
@@ -75,6 +75,11 @@ the binary verification command above.
 
 ## Build
 
+To quickly test that io.js was built correctly from any of the steps below:
```

Comment:
I wrestled with this myself, but I was afraid that leaving it at the bottom some people might miss it.

Copying it in each of the different OS instructions seems to be another option, but my thoughts are that it would make the readme too verbose.